crates/geoip/src/resolver.rs

```diff
@@ -0,0 +1,29 @@
+use std::{net::IpAddr, sync::Arc};
+
+pub use {
+    crate::maxminddb::geoip2::City,
+    local::LocalResolver,
```

Comment:
Ideally, should be importable from the outside as `geoip::LocalResolver` or `geoip::resolver::Local`
Same thing for the other one